metron-interface/metron-config/scripts/start-server-for-e2e.sh

```diff
@@ -14,10 +15,5 @@
 #  See the License for the specific language governing permissions and
 #  limitations under the License.
 #
-port: 4201
-
-rest:
-  host: localhost
-  port: 8080
-
-dirPath: alerts-ui
\ No newline at end of file
+SCRIPTS_ROOT="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+$SCRIPTS_ROOT/../node_modules/@angular/cli/bin/ng serve --proxy-config proxy.conf.json --aot $@
```

Comment:
For some reason, git recognized this change as a rename. It is actually a removal of a yml config file (belongs to our previous test server setup, no longer needed) and the addition of a shell script we used to fire up the UI locally.